config.json.example

```diff
@@ -4,16 +4,17 @@
     "stake_amount": 0.05,
     "dry_run": false,
     "minimal_roi": {
-        "60":  0.0,
-        "40":  0.01,
-        "20":  0.02,
-        "0":  0.03
+        "60": 0.0,
+        "40": 0.01,
+        "20": 0.02,
+        "0": 0.03
     },
     "stoploss": -0.40,
     "bid_strategy": {
         "ask_last_balance": 0.0
     },
-    "bittrex": {
+    "exchange": {
+        "name": "bittrex",
         "enabled": true,
```

Comment:
I think we can get rid of the `enabled` property because there can only be one exchange

Reply:
Done